terraform/container-app/functions.tf

```diff
@@ -6,8 +6,9 @@ resource "azurerm_storage_account" "function_storage" {
   account_replication_type = "LRS"
   tags                     = local.tags
```

Comment:
tfsec identifies a critical problem on this resource - wants to have `min_tls_version` specified as TLS1_2, but it is anyway so only semantic - could be fixed with the other ones, or as part of this as its the only critical?